course5/homework05/vector2d.cc

```diff
@@ -0,0 +1,106 @@
+#include "vector2d.h"
+#include <cmath> 
+
+using namespace vector2D;
+
+Vector2D::Vector2D()
+{
+	mX = 0.0;
+	mY = 0.0;
+}
+
+
+Vector2D::Vector2D(double x, double y)
```

Comment:
This you don't have to implement either; this is exactly the implementation that compiler would generate itself.